examples/transform/src/main/java/org/apache/apex/examples/transform/SimpleTransformApplication.java

```diff
@@ -19,43 +19,75 @@
 
 package org.apache.apex.examples.transform;
 
+import java.io.PrintStream;
 import java.util.HashMap;
 import java.util.Map;
 
+import org.apache.apex.malhar.lib.function.Function;
+import org.apache.apex.malhar.lib.function.FunctionOperator;
 import org.apache.hadoop.conf.Configuration;
 
+import com.google.common.annotations.VisibleForTesting;
+
 import com.datatorrent.api.Context;
 import com.datatorrent.api.DAG;
 import com.datatorrent.api.StreamingApplication;
 import com.datatorrent.api.annotation.ApplicationAnnotation;
 import com.datatorrent.common.partitioner.StatelessPartitioner;
-import com.datatorrent.lib.io.ConsoleOutputOperator;
 import com.datatorrent.lib.transform.TransformOperator;
 
 @ApplicationAnnotation(name = "TransformExample")
 /**
  * @since 3.7.0
  */
-public class Application implements StreamingApplication
+public class SimpleTransformApplication implements StreamingApplication
 {
+  @SuppressWarnings("serial")
+  public static final Function.MapFunction<Object, Void> CONSOLE_OUTPUT_FN = new Function.MapFunction<Object, Void>()
+  {
+    @Override
+    public Void f(Object input)
+    {
+      PrintStream out = System.out;
+      out.println(input);
+      return null;
+    }
+  };
+
+  @VisibleForTesting
+  Function.MapFunction<Object, ?> outputFn = CONSOLE_OUTPUT_FN;
+
+  private POJOGenerator pojoDataGenerator;
+
   @Override
   public void populateDAG(DAG dag, Configuration conf)
   {
-    POJOGenerator input = dag.addOperator("Input", new POJOGenerator());
+    pojoDataGenerator = dag.addOperator("Input", new POJOGenerator());
     TransformOperator transform = dag.addOperator("Process", new TransformOperator());
     // Set expression map
     Map<String, String> expMap = new HashMap<>();
     expMap.put("name", "{$.firstName}.concat(\" \").concat({$.lastName})");
     expMap.put("age", "(new java.util.Date()).getYear() - {$.dateOfBirth}.getYear()");
     expMap.put("address", "{$.address}.toLowerCase()");
     transform.setExpressionMap(expMap);
-    ConsoleOutputOperator output = dag.addOperator("Output", new ConsoleOutputOperator());
+    FunctionOperator.MapFunctionOperator<Object, ?> output = dag.addOperator("out",
+        new FunctionOperator.MapFunctionOperator<>(outputFn));
 
-    dag.addStream("InputToTransform", input.output, transform.input);
+    dag.addStream("InputToTransform", pojoDataGenerator.output, transform.input);
     dag.addStream("TransformToOutput", transform.output, output.input);
 
     dag.setInputPortAttribute(transform.input, Context.PortContext.TUPLE_CLASS, CustomerEvent.class);
     dag.setOutputPortAttribute(transform.output, Context.PortContext.TUPLE_CLASS, CustomerInfo.class);
     dag.setAttribute(transform, Context.OperatorContext.PARTITIONER, new StatelessPartitioner<TransformOperator>(2));
   }
+
+  public POJOGenerator getPojoDataGenerator()
```

Comment:
Perhaps make pojoGenerator `@VisibleForTesting` instead? Idea is to keep application code concise, yet allow for testing.

Reply:
done.